group/static/group/js/groupList.js

```diff
@@ -0,0 +1,37 @@
+'''
```

Comment:
QAQ. Please make use of dev tool.
![2015-05-12 12 23 15](https://cloud.githubusercontent.com/assets/6128538/7569707/1e9dc4ae-f83d-11e4-859e-ea934994e460.png)